### ISSUE TITLE
Habitat mapping in the NGOMEX over past and future environmental conditions

### ISSUE THREAD

**Issue by zenrabbit** [NCEAS/oss-2017#27]
Author: **Cassandra Glaspie**
Topic: habitat mapping

## Synthesis topic
I propose a synthesis project that will map habitat for a variety of species (decided by participants in the group) in the NGOMEX over past and future environmental conditions.  

## Data needs
Daily water temperature, salinity, and dissolved oxygen data will be obtained from a variety of sources (CTD/XBT casts, cruise data, etc.) for the years ~1950-2016 (subject to data availablility). We will also seek out scenarios of future environmental conditions (through 2100) using global or regional climate model runs. All code for processing environmental data will be produced in R statistical software, and all data will be organized into a relational database in an open source database management system. 

## Analytical approaches
This new database will be used to create spatial habitat suitability models for species of interest. These models may incorporate bioenergetics when available (growth rate potential modeling from my own work; Brandt et al. 1992) or may include a simple range of suitable environmental conditions for species with less available data. We can examine habitat suitability over time and space scales defined by the availability of environmental data. We will develop indices of habitat suitability from these data. This may include spatially and annually integrated averages, presence/absence of suitable habitat in a certain region, or location of the suitable habitat range centroid. 

## Impact/Significance
The outcomes of this research will be a quantitative understanding of suitable habitat for a variety of species throughout the water column in the NGOMEX. Past records of habitat suitability can be used to examine the impact of past climate on distribution of key species, and future habitat suitability can be used to develop a predictive framework for future distributions. As a group, we can look for commonalities or differences in expected distributions and use these to infer consequences of environmental change on ecosystem health and function. I expect this project will result in a publication, future collaborations, and a database that can be used for future projects, including my own research. This project fulfills several of the goals of the Gulf Research Initiative, including 1) providing a long-term, cross boundary perspective; 2) focusing on science to advance understanding; 3) synthesis and integration of GOM data; and 4) initiation of partnerships.

**Comment by Sunil2** [comment]
I like Cassandra Glaspie idea on this project. I am interested in this project. If location data are available for environmental variables, we can use “spatstat” package available on R-software to find the distribution pattern of those variables. Later, we can use “pcfcross” function to see the association between two variables. I think spatial distribution pattern and association will be helpful in a model development process. Just early thoughts...Cassandra, do you think identifying distribution and association of environmental variables are useful for habitat suitability model?

**Comment by demutsertlab** [comment]
Hi Cassie! Are you planning on using bioacoustic data (plus accompanying trawls) collected on the cruises in this project idea? Perhaps for validation of fish distribution if the habitat suitability models are based on environmental data? I think that may be a good way to get a handle on those data.

**Comment by ailich** [comment]
This project seems related to mine (#30). Projects #16 and #19 are also related to habitat suitability so there may be some potential to merge these projects.

**Comment by cnglaspie** [comment]
@demutsertlab I do have bioacoustic data for 2003-2011, and zooplankton samples/fish stomach contents from 2006-2008, and these can certainly be used for validation (I think this is on my "to do" list for my current funding anyways). @Sunil2 I am definitely interested in your spatial statistics expertise, and the spatial stats modules in the course. I am sure there are a lot of interesting things that can be done with the data I'm sitting on.

**Comment by demutsertlab** [comment]
I see a general interest emerge from this and several other proposals to map fish distributions and environmental factors, and link those in order to determine how environmental factors (water quality, habitat, disturbances, climate change) affect fish distribution. This aggregated synthesis project could result in habitat suitability indices, taxa-specific tolerance curves for environmental factors, and distribution maps (of fish populations and environmental factors) that could be used to validate output of spatial models. This would merge aspects of this one (#27) with #33 , #30 , #28 , #19 , #14 , #13 , and #10

**Comment by aesacco** [comment]
I agree with @demutsertlab that these projects could potentially be merged together. Aside from past/future habitat suitability and predictions, we should also look at how this can relate to the overall health of the Gulf of Mexico. Building baselines, especially with a synthesis project like this, could be important to other users and stakeholders of fisheries-based or ecosystem-based concerns.
